apache-hadoop-master/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM quay.io/ucsc_cgl/apache-hadoop-common:2.6.2
+FROM quay.io/ucsc_cgl/apache-hadoop-common
```

Comment:
Don't want to pin a version to pull from?

Reply:
Nope; look at the makefile. We build the latest `quay.io/ucsc_cgl/apache-hadoop-common` container prior to building this container. This makes it easier to update the two containers in tandem.